Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/WOfSImputation.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>maybe just add a line here saying.. eg 75% vs 90%, I know it's on the chart but helps :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>